packages/sonata-integration-bundle/CronJob/Admin/CronJobAdmin.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Draw\Bundle\SonataIntegrationBundle\CronJob\Admin;
+
+use Draw\Component\CronJob\Entity\CronJob;
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\AdminBundle\Route\RouteCollectionInterface;
+use Sonata\AdminBundle\Show\ShowMapper;
+
+/**
+ * @method CronJob getSubject()
+ */
+class CronJobAdmin extends AbstractAdmin
+{
+    protected function configureDatagridFilters(DatagridMapper $filter): void
+    {
+        $filter
+            ->add('id')
+            ->add('name')
+            ->add('command')
+            ->add('active');
+    }
+
+    protected function configureListFields(ListMapper $list): void
+    {
+        $list
+            ->addIdentifier('name')
+            ->add('command')
+            ->add('schedule')
+            ->add('active', null, ['editable' => true])
+            ->add('timeToLive')
+            ->add('priority')
+            ->add(
+                ListMapper::NAME_ACTIONS,
+                ListMapper::TYPE_ACTIONS,
+                [
+                    'actions' => [
+                        'show' => [],
+                        'edit' => [],
+                        'queue' => [
+                            'template' => '@DrawSonataIntegration/CronJob/CronJob/list__action_queue.html.twig',
+                        ],
+                        'delete' => [],
+                    ],
+                ]
+            );
+    }
+
+    protected function configureFormFields(FormMapper $form): void
+    {
+        $form
+            ->with('Definition', ['class' => 'col-md-8'])
+                ->add('name')
+                ->add(
+                    'command',
+                    null,
+                    [
+                        'help' => 'Enter the full command to run excluding stdOut and stdErr directive (... 2>&1 | logger -t ...)<p>Parameters bag is available. Use like %kernel.project_dir%</p>',
```

Comment:
Remove the The help about (... 2>&1 | logger -t ...) since we have remove the "decorator"